example/src/components/GetZoom.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { Text } from 'react-native';
+import MapboxGL from '@mapbox/react-native-mapbox-gl';
+
+import BaseExamplePropTypes from './common/BaseExamplePropTypes';
+import Page from './common/Page';
+import Bubble from './common/Bubble';
+
+class GetZoom extends React.Component {
+  static propTypes = {
+    ...BaseExamplePropTypes,
+  };
+
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      zoom: 9,
+    };
+
+
```

Comment:
Same here we can get rid of some of the white spaces

Reply:
done 😛